scripts/test/lint.py

```diff
@@ -94,6 +94,7 @@ class IncludesRequiredFieldsOnly(Lint):
         "description",
         "authors",
         "dependencies",
+        "tags",
```

Comment:
@sara-rn can you please document the `tags` field in the wiki: https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#nuspec ?